packages/wordpress-kit/src/lib/tailwindcssPlugin/components/Gallery.ts

```diff
@@ -0,0 +1,138 @@
+const getFigureStyles = (mobileScreen: string) => {
+  let styles = {
+    figure: {
+      width: 'calc(50% - 1rem);',
+      margin: '0',
+      display: 'flex',
+      flexGrow: '1',
+      position: 'relative',
+      maxWidth: '100%',
+      flexDirection: 'column',
+      alignItems: 'flex-start',
+    },
+  };
+
+  for (let column = 1; column <= 8; column++) {
+    styles = {
+      ...styles,
+      [`&.columns-${column}`]: {
+        figure: {
+          [`@media (min-width: ${mobileScreen})`]: {
+            width: `calc(${100 / column}% - 1rem) !important`,
```

Comment:
Can you add a comment here explaining what the `calc()` is doing?

Reply:
Sure it!